frontend/packages/console-dynamic-plugin-sdk/src/api/k8s-types.ts

```diff
@@ -50,3 +57,19 @@ export type K8sList = <R extends K8sResourceCommon>(options: {
 }) => Promise<R[]>;
 
 export type GetK8sResourcePath = (model: K8sModel, options: Options) => string;
+
+export type GetReference = (
+  group: string,
+  version: string,
+  kind: string,
+) => K8sResourceKindReference;
+
+export type GetReferenceForModel = (model: K8sModel) => K8sResourceKindReference;
+
+export type GetAPIVersionForModel = (model: K8sModel) => string;
+
+export type GetGroupVersionKindForResource = (resource: K8sResourceCommon) => K8sGroupVersionKind;
+
+export type GetGroupVersionKindForReference = (
```

Comment:
```suggestion
export type GetGroupVersionKindFromReference = (
```

Reply:
@rawagner at other places as well, we have followed the convention of `for` should we update all to `from` i.e

`GetGroupVersionKindForResource` to `GetGroupVersionKindFromResource`
`GetReferenceForModel` to `GetReferenceFromModel`
`GetAPIVersionForModel` to `GetAPIVersionFromModel`

let me know.

Reply:
hmm..on the second thought, I think the `for` is ok :)